iotdb-client/client-py/iotdb/Session.py

```diff
@@ -260,6 +238,33 @@ def init_connection(self, endpoint):
             self.__zone_id = self.get_time_zone()
         return SessionConnection(client, transport, session_id, statement_id)
 
+    def __get_transport(self, endpoint):
+        if self.__use_ssl:
+            import ssl
+            from thrift.transport import TSSLSocket
```

Comment:
Move the import port here for avoiding a log from thrift with python 3.12. 
https://github.com/apache/thrift/pull/3066/files